src/api/app/models/token/rebuild.rb

```diff
@@ -5,7 +5,7 @@ def self.token_name
 
   def call(options)
     # FIXME: Use the Package#rebuild? instead of calling the Backend directly
-    Backend::Api::Sources::Package.rebuild(package_from_association_or_params.project.name,
+    Backend::Api::Sources::Package.rebuild(project_from_association_or_params.name,
```

Comment:
most likely crashing in remote project case?

Reply:
It will never get here as `package_from_association_or_params` will be nil in the controller. You can't instantiate remote packages. It would also not make much sense to trigger a rebuild *for* a project that comes from remote right?

Reply:
```ruby
[6] pry(main)> Project.find_by(name: 'openSUSE.org').remoteurl
  Project Load (1.3ms)  SELECT `projects`.* FROM `projects` WHERE `projects`.`id` != 0 AND `projects`.`name` = 'openSUSE.org' LIMIT 1
=> "https://api.opensuse.org/public"
[7] pry(main)> Package.get_by_project_and_name('openSUSE.org:home:hennevogel', 'ponysay')
Trying to find local project openSUSE.org:home, remote_project hennevogel
  Project Load (1.1ms)  SELECT `projects`.* FROM `projects` WHERE `projects`.`id` != 0 AND `projects`.`name` = 'openSUSE.org:home' LIMIT 1
Trying to find local project openSUSE.org, remote_project home:hennevogel
  Project Load (0.8ms)  SELECT `projects`.* FROM `projects` WHERE `projects`.`id` != 0 AND `projects`.`name` = 'openSUSE.org' LIMIT 1
Found local project openSUSE.org for home:hennevogel with remoteurl https://api.opensuse.org/public
=> nil
```